graphics/GraphicsMagick/Portfile

```diff
@@ -38,12 +39,15 @@ depends_lib             port:libxml2 \
                         port:lcms2 \
                         port:jasper \
                         port:jpeg \
-                        port:webp
+                        port:webp \
+                        path:lib/libomp/libomp.dylib:libomp
 
 use_xz                  yes
 
 # llvm-gcc-4.2 gives "Undefined symbols for architecture x86_64: ___builtin_object_size"
-compiler.blacklist      *llvm-gcc-4.2
+# OpenMP is very beneficial for GM: http://www.graphicsmagick.org/OpenMP.html
+# (mp-clang.3.[8..] support openmp)
+configure.compiler      macports-clang-7.0
```

Comment:
Don't mandate a particular compiler. Instead, add all unsuitable compilers to `compiler.blacklist`. If all compilers in the default list are unsuitable, append a suitable compiler to `compiler.fallback`.

Reply:
I concur. Macports clang 5.0 is I believe the default fallback and if it is sufficient for openmp support should be allowed.